src/NumericalAlgorithms/DiscontinuousGalerkin/MetricIdentityJacobian.cpp

```diff
@@ -302,63 +314,6 @@ void metric_identity_jacobian_quantities(
          ++storage_index) {
       (*jacobian)[storage_index] *= get(*det_jacobian);
     }
-  } else {
-    {
-      // In 3d det(J inv_jac) = 1/J^2.
```

Comment:
Can you re-add this particular comment line between new lines 305, 306? This will I think help clarify the sqrt in 3d.